### ISSUE TITLE
Kubeflow install on AWS existing cluster stuck at setup screen

### ISSUE THREAD

**Issue by acesir** [kubeflow/kubeflow#4648]
/kind bug

**What steps did you take and what happened:**
I have followed the steps for installing k8s_istio.0.7.0 here https://www.kubeflow.org/docs/started/k8s/overview/ on an existing cluster. The steps I have followed work perfectly fine in GCP, kubeflow is running and I have been able to run my own pipelines but in AWS I am running into an issue where the startup page cannot be bypassed regardless of having the profiles created manually or through the UI itself

**What did you expect to happen:**
Expected to be able to create a namespace on the startup screen

**Anything else you would like to add:**
Mentioning again that everything worked correctly for deploying on an existing GCP cluster while on AWS all the services are deployed and working but the startup page will not let me login after the namespace and profiles have been created.

Here is some inspect data from the startup webpage is:

{hasAuth: false, user: "anonymous", hasWorkgroup: false}
hasAuth: false
hasWorkgroup: false
user: "anonymous"

{user: "anonymous@kubeflow.org",…}
isClusterAdmin: true
namespaces: []
platform: {kubeflowVersion: "unknown", provider: "other://", providerName: "other"}
user: "anonymous@kubeflow.org"

Not sure why namespaces is []empty while the namespace and profile for anonymous is created. I would also like to add that on the startup page it notifies that the namespace is created even though it never allows for logging in:

<img width="545" alt="Screen Shot 2020-01-13 at 3 51 17 PM" src="https://user-images.githubusercontent.com/5704659/72295160-95e4b780-361c-11ea-9ef4-bc83eb9d2c5a.png">

**Environment:**
AWS

- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard):
- kfctl version: kfctl v0.7.0
- Kubernetes platform: AWS EKS 
- Kubernetes version: 1.14
- OS:

**Comment by descampsk** [comment]
Try to delete the profil : kubectl delete profiles -n kubeflow anonymous
And retry

**Comment by yanniszark** [comment]
Hi @acesir!
The `k8s_istio` config doesn't provide authentication and it seems that there is something wrong with the way the CentralDashboard handles that.
cc @krishnadurai @avdaredevil 

In the meantime, I would suggest trying out the existing_arrikto config for v0.7, which includes authentication and shouldn't present this issue:
https://www.kubeflow.org/docs/started/k8s/kfctl-existing-arrikto/

**Comment by acesir** [comment]
@yanniszark I understand there is no authentication setup with this install but that should no prevent it from being unusable or functional. As it stands now I should at least be able to get to kubeflow UI and select namespaces/profiles created. As I mentioned above, the same process works on GCP consistntly but no so on AWS.

Thanks

**Comment by yanniszark** [comment]
> @yanniszark I understand there is no authentication setup with this install but that should not prevent it from being unusable or functional.

Agreed, this is likely a bug.
I am suggesting an approach to get started in the meantime, so you don't have to wait for the issue to be resolved in order to use Kubeflow :)

**Comment by acesir** [comment]
> > @yanniszark I understand there is no authentication setup with this install but that should not prevent it from being unusable or functional.
> 
> Agreed, this is likely a bug.
> I am suggesting an approach to get started in the meantime, so you don't have to wait for the issue to be resolved in order to use Kubeflow :)

@yanniszark  Understood I will try that for now and can update this issue if it indeed does resolve in AWS with https://www.kubeflow.org/docs/started/k8s/kfctl-existing-arrikto/

**Comment by acesir** [comment]
@yanniszark just to update. I used https://www.kubeflow.org/docs/started/k8s/kfctl-existing-arrikto/ and after logging with default credentials admin@kubeflow.org i am getting the same startup page to create a namespace. Same issue as above, namespace and profile get created but I cannot get past the setup screen.

This seems to be an issue with Profiles in AWS EKS possibly? Not sure if you have any ideas to get past this.

**Comment by krishnadurai** [comment]
@acesir could you please paste the output of

`kubectl get profiles`

and 

Could you also post the HAR file from the browser (navigate to tools -> network tab) which you are using?

**Comment by yanniszark** [comment]
To add to what @krishnadurai asked for, can you also provide the logs of the Central Dashboard deployment?

**Comment by acesir** [comment]
**profiles output:** 
```
❯ kubectl get profiles
NAME    AGE
admin   9h
❯ kubectl describe profile admin
Name:         admin
Namespace:
Labels:       <none>
Annotations:  kubectl.kubernetes.io/last-applied-configuration:
                {"apiVersion":"kubeflow.org/v1beta1","kind":"Profile","metadata":{"annotations":{},"name":"admin"},"spec":{"owner":{"kind":"User","name":"...
API Version:  kubeflow.org/v1beta1
Kind:         Profile
Metadata:
  Creation Timestamp:  2020-01-22T02:49:35Z
  Generation:          1
  Resource Version:    12203
  Self Link:           /apis/kubeflow.org/v1beta1/profiles/admin
  UID:                 d3309b92-3cc1-11ea-8dae-0280ce865bbc
Spec:
  Owner:
    Kind:  User
    Name:  admin@kubeflow.org
Events:    <none>
```
**HAR file converted to .txt:**
[istio-ingressgateway.istio-system.svc.cluster.local.txt](https://github.com/kubeflow/kubeflow/files/4097212/istio-ingressgateway.istio-system.svc.cluster.local.txt)

**Central dashboard logs:**
[logs-from-centraldashboard-in-centraldashboard-546fc5bf8b-6rhls.txt](https://github.com/kubeflow/kubeflow/files/4097216/logs-from-centraldashboard-in-centraldashboard-546fc5bf8b-6rhls.txt)

I tried creating various different profiles so you might see various different namespace names. But it seems a lot of forbidden errors appear in the admin namespace that was created as part of the profile.

**Comment by acesir** [comment]
Update:

I moved away from creating an EKS cluster and deployed using KOPS. The issue does not exist with a non managed kubernetes cluster and seems to be isolated to EKS only. I can login just fine and the profile/namespace is created. The only issue i see is the admission-webhook pod seems to not be coming up and is showing an error : **MountVolume.SetUp failed for volume "webhook-cert" : secret "webhook-certs" not found**

I am deploying kubeflow 0.7 so not sure if this is resolved in 0.7.1
